Classes/CDSRC/Libraries/Translatable/Domain/Model/AbstractTranslation.php

```diff
@@ -40,7 +40,7 @@
  *
  * @author Matthias Toscanelli <m.toscanelli@code-source.ch>
  */
-abstract class AbstractTranslation
+abstract class AbstractTranslation implements TranslationInterface
```

Comment:
This was missing previously but it looks like it was working.
@codesource can you please confirm we need to implement this interface here?